### ISSUE TITLE
Checkable right click Context Menu

### ISSUE THREAD

**Issue by Christian-SWS** [CNFeffery/feffery-antd-components#117]
Dear Feffery-antd-components community,

I'm working on creating a right click context menu wihtin my Dash App, which works fine due to the great help of @CNFeffery.

Example Code can be found here: 
https://github.com/CNFeffery/feffery-antd-components/issues/115

I was now wondering if there is a possiblity to be able to have some options within the right click context menu which are "checkable". I found the AntdTree component which is checkable but I can't use the "freePosition" option for this, which is important fo me to access the context menu from wherever I right click within the FefferyDiv.

Did I miss a Feffery function which might provide this "checkability"?

**Comment by CNFeffery** [comment]
@Christian-SWS As a test feature in the pre-release version of fac 0.3.x, child items in AntdDropDown can be selected, and multiple selection is supported ( install fac >= 0.3.0a1 ):
![image](https://github.com/CNFeffery/feffery-antd-components/assets/49147660/fc8140d9-05b8-4769-ad98-aca52fdddb5f)

```python
import json
import dash
from dash import html
import feffery_utils_components as fuc
import feffery_antd_components as fac
from dash.dependencies import Input, Output

app = dash.Dash(__name__, suppress_callback_exceptions=True)

app.layout = html.Div(
    [
        fuc.FefferyDiv(
            'Dropdown Area',
            id='dropdown-free-position-demo-trigger',
            enableListenContextMenu=True,
            style={
                'borderRadius': 6,
                'background': '#adb5bd',
                'height': 400,
                'color': 'white',
                'display': 'flex',
                'justifyContent': 'center',
                'alignItems': 'center'
            }
        ),

        html.Div(id='dropdown-free-position-demo-container'),
        html.Pre(
            id='show-event'
        )
    ],
    style={
        'padding': 50
    }
)


@app.callback(
    Output('dropdown-free-position-demo-container', 'children'),
    Input('dropdown-free-position-demo-trigger', 'contextMenuEvent'),
    prevent_initial_call=True
)
def dropdown_free_position_demo(contextMenuEvent):

    if contextMenuEvent:

        context = fac.AntdDropdown(
            id='current-dropdown',
            visible=True,
            freePosition=True,
            freePositionStyle={
                'left': contextMenuEvent['clientX'],
                'top': contextMenuEvent['clientY']
            },
            menuItems=[
                {
                    'title': f'Option{i}',
                    'key': f'key{i}'
                }
                for i in range(1, 6)
            ],
            trigger='click',
            selectable=True,
            multiple=True
        )

        return context


@app.callback(
    Output('show-event', 'children'),
    Input('current-dropdown', 'selectedKeys'),
    prevent_initial_call=True
)
def collect_dropdown_event(selectedKeys):

    if selectedKeys:
        return json.dumps(
            dict(selectedKeys=selectedKeys),
            indent=4,
            ensure_ascii=False
        )

    return dash.no_update


if __name__ == '__main__':
    app.run(debug=True)

```

**Comment by Christian-SWS** [comment]
Dear CNFeffery, thanks again for this fast response! 
I installed the pre-release version and it works fine! My 'last' problem is now, that the whole context menu content is now selectable. I need to have selectable and non selectable options within the context menu. Do I have to manually remove the 'keys' which I dont want to be selectable from the 'selectedKeys' list oris there a more elegant way of having selectable and non selectable options within the context menu?

Thank you again for your great support!

**Comment by CNFeffery** [comment]
@Christian-SWS are there some other requirements? I can add those new features for `AntdDropdown` in the next alpha preview version.

**Comment by CNFeffery** [comment]
@Christian-SWS available from fac 0.3.0a3, use `nonSelectableKeys` in `AntdDropdown` to specific keys that should not be selected.

**Comment by Christian-SWS** [comment]
@CNFeffery thanks again! It works perfectly!
